docs/lib/datastore/fragments/flutter/datastore-events.md

```diff
@@ -0,0 +1,23 @@
+```dart
+AmplifyDataStore.events.listenToDataStore((event) {
+    if (event["eventName"] == "networkStatus") {
+        setState(() {
+            networkIsUp = event["active"];
+        });
+    }
+});
+```
+
+<amplify-callout>
+
+Please note that all amplify-flutter DataStore Hub events are currently structured as serialized maps.  The event name is specified in an 'eventName' attribute within the map.
+
+Support for custom types representing these events as well as automated instantiation of models within the event payloads is forthcoming.  
```

Comment:
Edit for conciseness

Coming soon: Support for custom event types and automated model instantiation within the event payload.

(maybe add a github link fro tracking)

Reply:
Added